frontend/app_flowy/packages/appflowy_editor/lib/src/render/emoji/emoji_select_widget.dart

```diff
@@ -0,0 +1,187 @@
+import 'package:appflowy_editor/src/core/document/node.dart';
+import 'package:appflowy_editor/src/editor_state.dart';
+import 'package:appflowy_editor/src/infra/flowy_svg.dart';
+import 'package:appflowy_editor/src/render/selection_menu/selection_menu_service.dart';
+import 'package:appflowy_editor/src/render/style/editor_style.dart';
+import 'package:flutter/material.dart';
+import 'package:app_flowy/workspace/presentation/widgets/emoji_picker/src/emoji_picker.dart';
+import 'package:appflowy_editor/appflowy_editor.dart';
```

Comment:
Don't import `'package:appflowy_editor/appflowy_editor.dart';`. Just import the necessary files.